frontend/src/setupTests.js

```diff
@@ -1,11 +1,115 @@
-import {
-  mockApolloClient,
-  mockLoadScript,
-  mockGoogleAuth,
-  mockSentry
-} from "./utils/testing";
+export function mockApolloClient() {
+  // Similar to MockedProvider, except Apollo doesn't provide a native
+  // client mock out of the box.
+  //
+  // This exposes an addMock function on the apolloClient, which takes the same
+  // input as a mock entry in a MockedProvider allows.
+  jest.mock("./utils/apollo", () => {
```

Comment:
they should really document MockLink better here as that + ApolloProvider is a much better strategy than MockedProvider